### DISCUSSION TITLE
有些接口显示400了，但没改过代码

### DISCUSSION THREAD

**Discussion by logWC** [Binaryify/NeteaseCloudMusicApi#1538] [Q&A]
浏览器转到报错url上，显示如下：

{"code":-462,"data":{"actionCode":"1048401","verifyType":40,"verifyId":1007602,"verifyUrl":"https://st.music.163.com/encrypt-pages","blockText":"验证成功后，可进行下一步操作哦~","verifyToken":"00.40.7af7648233b89664186c091e7b4e9098.3549119416","btnText":"","orpheusUrl":"","frontRuleIds":[1056402,1078401,1048401],"params":{"event_id":"00237808556640079886","sign":"aacb74c037e9bf47cab45990d1874e88"},"url":"https://st.music.163.com/encrypt-pages?params=%7B%22sign%22%3A%22aacb74c037e9bf47cab45990d1874e88%22%2C%22event_id%22%3A%2200237808556640079886%22%7D&verifyType=40&verifyId=1007602&verifyToken=00.40.7af7648233b89664186c091e7b4e9098.3549119416&actionCode=1048401"},"message":"请完成验证操作"}

**Comment by logWC** [comment]
好像是有一个安全认证，但具体还不知道怎么写

**Reply by surfman010** [reply]
你好，这个问题解决了吗，我也碰到这个问题了，浏览器端调用[获取音乐 url]这个接口会返回这个，手机端调用好像不会，而且之前使用的https://music.163.com/song/media/outer/url?id=id.mp3 这个地址当歌曲url地址是可以用的，现在报错Cross-Origin Read Blocking (CORB) 已屏蔽 MIME 类型为 application/json 的跨域响应，但是链接放在浏览器单独打开是没问题的可以播放。

**Reply by sometimeQ** [reply]
> 你好，这个问题解决了吗，我也碰到这个问题了，浏览器端调用[获取音乐 url]这个接口会返回这个，手机端调用好像不会，而且之前使用的https://music.163.com/song/media/outer/url?id=id.mp3 这个地址当歌曲url地址是可以用的，现在报错Cross-Origin Read Blocking (CORB) 已屏蔽 MIME 类型为 application/json 的跨域响应，但是链接放在浏览器单独打开是没问题的可以播放。

大佬，请问怎么解决的

**Comment by Binaryify** [comment]
看看新版本是不是修复好了
